### DISCUSSION TITLE
How meetings register their meeting

### DISCUSSION THREAD

**Discussion by asmmargod** [code4recovery/12-step-meeting-list#1216] [General]
Hi. I wanted to find out how a meeting can register their meeting. There does not seem to be a form for a new meeting to submit and looking at the documention I'm struggling to find an overlook of how this process of registering/updating meetings works etc. Can someone point me to right direction?

**Comment by kiyote33** [comment]
Hi @asmmargod,
If you haven't reviewed our documentation yet, you can do so by visiting [https://wordpress.org/plugins/12-step-meeting-list/](https://wordpress.org/plugins/12-step-meeting-list/).  There are multiple ways a site administrator can setup their site to input new meetings:
1. Admins can input meetings directly, one meeting at a time using the Meetings/Add New WordPress Admin menu item.
2. Admins can import a list of meetings (which meet GSO's Meeting Guide Spec) using the Meetings/Import & Export WordPress Admin menu item. Check out our [How To videos here](https://code4recovery.org/how-to) for more insight on doing this.
3. If your site is using the Legacy UI user interface display option, you can add the 12 Step Meeting List Feedback Enhancement plugin from the WordPress directory to display Change/New/Remove request forms to your site. See the [How To video here](https://code4recovery.org/how-to/how-to-clarify-meeting-feedback). 
4. If your site is using the TSML UI user interface display option, you can choose to link to a form plugin like gravity forms with all the fields filled out like the above enhanced feedback utility.  Here's the [link on how to do ](https://wordpress.org/plugins/12-step-meeting-list/#can%20i%20set%20a%20custom%20%22update%20meeting%20info%22%20button%20in%20tsml%20ui%3F) this. Once you create the form, you use the code supplied in the FAQ to force the button to link to that form.

Hope this helps. Let me know if I can be of further assistance.

**Reply by virtualtom** [reply]
Is this still valid?  I've added the url for my feedback form into multiple functions.php files and the update button still wants to send an email.  Am I missing a step, or location of the functions.php it needs to go in?

**Reply by joshreisner** [reply]
hi @virtualtom this should just be the `functions.php` for your active wordpress theme. i believe you then need to update a meeting for it to apply this URL to your meetings. let us know if that doesn't work (and send us your website url). thanks!

**Reply by virtualtom** [reply]
Is there a specific place it needs to be in the functions.php, or anything that needs to be added with it?  The line is there, and I did an import to the plugin and still no change.  URL is https://buffaloaany.org

**Reply by joshreisner** [reply]
ah, there may be a gap here and that it doesnt apply the url when importing. for now go to a meeting and click the update button and let us know if theres a change. thanks

**Reply by virtualtom** [reply]
I may have put it in the wrong file, my provider has an old folder from before the website crashed last year and I think when they fixed it the themes got split between folders.  Anyway adding it to both and doing the update on a meeting did it.  Any plans on adding this into the plugin in case people change themes without adding this to the functions.php in the new theme?

**Reply by dougie-wougie** [reply]
I added the WPforms and added the $tsml_feedback_url = '/meeting-update'; and it works, but Josh's mentioned earlier updating the one meeting. How do I make this change for all the meetings? Should I do an export/import to get all meetings to use the new $tsml_feedback_url working on all meetings?

**Reply by virtualtom** [reply]
Josh let me know that it doesn't work with the import, had to click update on one meeting. After that it worked after a refresh of the page.

**Reply by dougie-wougie** [reply]
So, updating one meeting makes it work for all meetings. Cool.
